### PR TITLE
AB-10 auto-close modal and refresh after deletion

### PR DESCRIPTION
**Pull request by stianlaa**
Relevant changes are in Explorer.vue, the other are largely due to linting changes.

[AB#10](https://dev.azure.com/incrementally/6e119c52-39a8-4798-85c7-142aed46bee9/_workitems/edit/10)

### CODE REVIEW

File: src/layouts/Explorer.vue [hunk 1]
```diff
@@ -121,7 +168,12 @@ export default class Explore extends Vue {
       )
       .then(res => res.json())
       .then(json => (this.entries = json))
-      .then(() => this.searching = false);
+      .then(() => (this.searching = false));
```
Comment (holwech) [inline src/layouts/Explorer.vue:171]: I suggest that we stick with the async/await pattern as much as possible (I haven't been so good at being consistent with this myself), as it looks nicer and is more modern.

File: src/models/RecordingEntry.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
 export default interface IRecordingEntry {
     id: string;
     recording: string;
-};
+};;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
```
Comment (holwech) [inline src/models/RecordingEntry.ts:4]: We should probably find out what's going on with the linting here 🤔

**Comment by holwech** [comment]
Looks good! Let's merge with the linting and then make sure that we always lint before a PR (again I haven't been so consistent with this myself).

File: src/models/RecordingEntry.ts (hunk 2 above)
Comment (stianlaa) [reply src/models/RecordingEntry.ts:4]: ow, I didn't see this, just ran the linting before comitting. Fixing.

Have you considered just using the ESLint extension on VScode, and have the onsave hook enabled? Works well for me.

File: src/layouts/Explorer.vue (hunk 1 above)
Comment (stianlaa) [reply src/layouts/Explorer.vue:171]: Hm, yeah, good point, converting to async/await pattern.

Interesting short discussion on the subject here: https://stackoverflow.com/questions/59430432/which-is-better-await-vs-then-for-nodejs-promise-and-how-await-works 
TLDR: Improved performance and readability

File: src/models/RecordingEntry.ts (hunk 2 above)
Comment (holwech) [reply src/models/RecordingEntry.ts]: I've had issues with this comma thing before when not using the that extension, so there seems to be a bug in ESLint somehow.
